README.md

```diff
@@ -147,7 +147,7 @@ This is Day to Day Work Linux Cheatsheet for Software Engineers.
 |Sleep Program| CTRL + z <correct>
 |Search History| CTRL + r <correct>
 |Repeat Last Command| !! <correct>
```

Comment:
|Sleep Program| CTRL + z <correct>
|Search History| CTRL + r <correct>
|Repeat Last Command| !! <correct>

You should remove the correct blocks? 

These should be as below:

Sleep Program| CTRL + z
|Search History| CTRL + r
|Repeat Last Command| !!